### PR TITLE
create popup-window

### PR DESCRIPTION
**Pull request by mohamed20163858**
create popup-window for both desktop and mobile versions

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -654,6 +790,88 @@ header a {
...
+    justify-self: flex-end;
+  }
+
+  .project-pop-up-window .close-item {
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    min-height: 15rem;
+    position: relative;
+    grid-area: photo;
+  }
```
Comment (1GORDON) [inline css/main.css:833]: Great design @mohamed20163858 🤝

- [ ] You may kindly fix your popup images as they appear to be stretched. You may increase their height and make them take up most of the space on the popup windows just as they appear in the Figma design.
- Your design
desktop image

- Figma design
Figma desktop image

File: css/main.css [hunk 2]
```diff
@@ -420,6 +432,105 @@ header a {
...
+
+  .project-pop-up-window ul {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+  }
```
Comment (1GORDON) [inline css/main.css:500]: Great job @mohamed20163858 🤝

- [ ] You may also change the color of the closing icon to your mobile popup. As required in the project requirements we are supposed to close using this button to return to the main page but unfortunately, it is not visible enough. You could change it to a darker color.
cancel button

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mohamed20163858 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️ 
- Descriptive Pull Request ✔️
- Linters are passing ✔️
- Professional Readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/main.css (hunk 1 above)
Comment (mohamed20163858) [reply css/main.css]: @1GORDON thanks  a lot for mentioning that i already resolved this problem

File: css/main.css (hunk 2 above)
Comment (mohamed20163858) [reply css/main.css:500]: @1GORDON actually there was a different background image with a different close button which was white but the problem here is in my javascript file there was no event listener for resizing popup window manually as it was totally static i solved it by adding an event listener to  detect any resize change and change the styling depend on the width dimension belong to mobile or desktop thank again for mention that

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @mohamed20163858 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**